### PR TITLE
Fix for running without Gx

### PR DESCRIPTION
**Pull request by himanshu3**
Added fix for following items:
1) Fix for running without Gx
2) Corrected decoding of PCO, APCO, EPCO IEs.
3) Added OI flag to check SGW relocation case.

### CODE REVIEW

**Comment by himanshu3** [comment]
> Changes look good.
> Please attach the test result and proceed.

Attaching CP log file describing successfull attach-detach of single UE.
[cp_2020-06-02_14-37.log](https://github.com/omec-project/ngic-rtc/files/4717856/cp_2020-06-02_14-37.log)

File: libgtpv2c/src/gtp_ies_decoder.c [hunk 1]
```diff
@@ -371,7 +371,7 @@ int decode_gtp_prot_cfg_opts_ie(uint8_t *buf,
     uint16_t decoded = 0;
     value->pco = decode_bits(buf, total_decoded, 8, &decoded);
     total_decoded += decoded;
-    return total_decoded/CHAR_SIZE;
+	return value->header.len + IE_HEADER_SIZE;
```
Comment (thakurajayL) [inline libgtpv2c/src/gtp_ies_decoder.c:374]: PCO decoding does not seem to be correct. PCO is 256 byte array length field.  currently we are just copying 1 byte. Same applies to other IEs.

Comment (himanshu3) [reply libgtpv2c/src/gtp_ies_decoder.c:374]: The correct structure and encoding/decoding of PCO, APCO will be part of separate libgtpv2c project.

**Comment by pingping-lin** [comment]
retest this please

**Pull request merged**